content/news/march-2016-tournament-annoucment.md

```diff
@@ -0,0 +1,22 @@
+
+
+[March 2016 OpenRA Tournament](..\images\news\march-2016-tournament.jpg)
+
+If you are interested in some heated tag-team battles in OpenRA [Register Your Team Here] (http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19414)!  
+
+March 5th-6th and 11th - 12th 2016!
```

Comment:
You could say "It will run from" and the spacing between the days and the hyphens is inconsistent.